components/automate-ui/src/app/page-components/event-icon/event-icon.component.ts

```diff
@@ -20,6 +20,7 @@ export class EventIconComponent {
       case 'node': return 'storage';
       case 'cookbook': return 'chrome_reader_mode';
       case 'version': return 'chrome_reader_mode';
+      case 'cookbook_artifact_version': return 'chrome_reader_mode';
```

Comment:
Adds the cookbook icon and not the question mark icon. 
before
![image](https://user-images.githubusercontent.com/1679247/85806617-65916100-b704-11ea-9c98-6b51e015422f.png)
After
![image](https://user-images.githubusercontent.com/1679247/85806638-7a6df480-b704-11ea-97b4-53a0430850e4.png)